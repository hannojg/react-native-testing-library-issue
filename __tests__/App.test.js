import React from "react";
import App from "./../App";
import { render } from '@testing-library/react-native';

it("should render correctly", () => {
    const tree = render(<App />);
    expect(tree.toJSON()).toMatchSnapshot();
})