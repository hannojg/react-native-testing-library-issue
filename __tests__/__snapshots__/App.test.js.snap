// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  0
</View>
`;
